index.md

```diff
@@ -2,25 +2,58 @@
 title: "R-multiverse"
 ---
 
-<img src="logo/logo-hexbin.png" align="right" width="140" />
+R-multiverse is a general-purpose R package repository system with
+[transparent governance](policies.md) and open source infrastructure.
+R-multiverse provides:
 
-#### A community-curated package repository system, powered by <br />R-universe
+1. A home for packages that fall outside the
+[CRAN Repository Policy](https://cran.r-project.org/web/packages/policies.html).
```

Comment:
outside the scope of other repositories such as CRAN or Bioconductor?

Reply:
added in da7b8642c27974b6e25d50ee547489971d0b5a73